ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/PauseButton.tsx

```diff
@@ -135,7 +136,7 @@ export default function PauseFunctionButton({ functionSlug, disabled }: PauseFun
     return null;
   }
 
-  const isPaused = !fn.current && !fn.archivedAt;
+  const { isPaused } = fn;
```

Comment:
Should this logic also be replaced? https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/%5BenvironmentSlug%5D/functions/%5Bslug%5D/layout.tsx#L40